### ISSUE TITLE
Uploading notification gets stuck on Q

### ISSUE THREAD

**Issue by oakkitten** [Rouji/Hupl#14]
often i can see two notifications after download has finished:

    Complete: file.jpg
    https://url
    <image>

    file.jpg     8 KiB/80 KiB
    ▰▰▱▱▱▱▱▱▱▱▱▱▱▱

the second one vanishes after a few seconds, but it's still annoying

**Comment by Rouji** [comment]
Yeah I've noticed some slight weirdness with notifications, even on O. Should probably make some time to look at those. :thinking:

**Comment by Rouji** [comment]
I don't remember explicitly fixing this, but it's not been happening anymore afaict
